tests/unit/remote/test_base.py

```diff
@@ -69,6 +70,7 @@ def test_hashes_exist(object_exists, traverse, dvc):
 @mock.patch.object(ObjectDB, "_path_to_hash", side_effect=lambda x: x)
 def test_list_hashes_traverse(_path_to_hash, list_hashes, dvc):
     odb = ObjectDB(FileSystem(), None)
+    odb.fs.TRAVERSE_PREFIX_LEN = 3
```

Comment:
Need to figure out why this is failing for TRAVERSE_PREFIX_LEN=2.

Reply:
For the record: we'll likely get rid of it in favor of `fs.find("12/3*")` or something.